src/app/office/projects/detail/responses/responses.component.scss

```diff
@@ -2,4 +2,17 @@
   &__bar {
     margin-bottom: 12px;
   }
+
+  &__no-responses {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    align-items: center;
+    margin-top: 68px;
+    text-align: center;
+
+    & span {
+      color: var(--dark-grey);
```

Comment:
The color is right. But it's not specified in figma, write comment in figma and tag Polina to resolve it